src/Shaders/default.fragment.fx

```diff
@@ -456,8 +459,17 @@ color.rgb = max(color.rgb, 0.);
     #endif
 #endif
 
-#if !defined(PREPASS) || defined(WEBGL2) 
+#if !defined(PREPASS) || defined(WEBGL2)
 	gl_FragColor = color;
 #endif
 
+#if ORDER_INDEPENDENT_TRANSPARENCY
+	if (fragDepth == nearestDepth) {
+		frontColor.rgb += color.rgb * color.a * alphaMultiplier;
+		frontColor.a = 1.0 - alphaMultiplier * (1.0 - color.a);

```

Comment:
I am having trouble to match this and the equations from under blending in https://developer.download.nvidia.com/SDK/10.5/opengl/src/dual_depth_peeling/doc/DualDepthPeeling.pdf Could you explain it ?

I have the same comment for the rest of the blending equations in use. Would be great to better understand the setup on my side :-)